quesma/clickhouse/clickhouse_test.go

```diff
@@ -577,16 +565,16 @@ func TestJsonConvertingBoolToStringAttr(t *testing.T) {
 		},
 	}
 
-	attrs, others, err := BuildAttrsMapAndOthers(m, config)
+	attrs, err := BuildAttrsMapAndOthers(m, config)
 	assert.NoError(t, err)
-	assert.Equal(t, 0, len(others))
 	assert.Equal(t, 3, len(attrs))
 	for k := range attrs {
 		assert.Contains(t, k, "string")
 	}
 }
 
 func TestCreateTableString_1(t *testing.T) {
+	t.Skip("Others has been removed")
```

Comment:
Maybe remove this test altogether then?

Reply:
You are right. I left it just to revisit this decision later